runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -219,11 +219,11 @@ def SanityCheck(): bool
   var is_check_ok = true
   # Need either the +terminal feature or +channel and the prompt buffer.
   # The terminal feature does not work with gdb on win32.
-  if (way ==# 'prompt') && !has('channel')
+  if (way is Way.Prompt) && !has('channel')
```

Comment:
By the way, I found `==`  for enum is incorrect, it may be an internal bug in vim9script. so I use `is` in this PR. See https://github.com/vim/vim/issues/15067